.pre-commit-config.yaml

```diff
@@ -1,22 +1,22 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Looks like we could also include prettier as a pre-commit hook.

See: https://prettier.io/docs/en/precommit.html#option-3-pre-commithttpsgithubcompre-commitpre-commit